test_integration/m1_facilitator/utils.ts

```diff
@@ -79,7 +79,7 @@ export default class Utils {
   public static async waitForCondition(
     boolFunction: Function,
     intervalTime: number = 2000,
-    maxInterval: number = 60,
+    maxInterval: number = 100,
```

Comment:
How increasing maxInterval is useful here?

Reply:
It's the max interval/time before assertions. It's helpful when graph node is slow.